src/System.Text.Encoding/tests/Encoder/EncoderConvert2.cs

```diff
@@ -188,6 +188,145 @@ public void PosTest9()
             VerificationHelper(encoder, chars, 0, 1, bytes, 0, bytes.Length, false, 1, 2, true, "009.3");
             VerificationHelper(encoder, chars, 0, 1, bytes, 0, bytes.Length, true, 1, 2, true, "009.4");
         }
+        
+        // PosTest10: Call Convert to convert a arbitrary character array with ASCII encoder
+        [Fact]
+        public void PosTest10()
+        {
+            char[] chars = new char[c_SIZE_OF_ARRAY];
+            byte[] bytes = new byte[c_SIZE_OF_ARRAY];
+            Encoder encoder = Encoding.ASCII.GetEncoder();
+
+            for (int i = 0; i < chars.Length; ++i)
+            {
+                chars[i] = _generator.GetChar(-55);
+            }
+
+            int charsUsed;
+            int bytesUsed;
+            bool completed;
+            encoder.Convert(chars, 0, chars.Length, bytes, 0, bytes.Length, false, out charsUsed, out bytesUsed, out completed);
```

Comment:
>false [](start = 76, length = 5)

could you please write this as flush: false for the sake of readability? also apply this to the rest of changes.